tests/5.Transaction.js

```diff
@@ -333,4 +335,238 @@ describe('Sign', function () {
 
         });
     });
+
+    describe('transaction making functions', function () {
+        it('should be able to use helper to make a payment transaction', function() {
```

Comment:
It would better if these tests would compare to golden binaries. but it can be sufficient if `algosdk.makePaymentTxn` does it already.

Reply:
:+1: Asset transaciton goldens are in `7.AlgoSDK.js`